src/components/typography/text/text.js

```diff
@@ -10,6 +10,7 @@ const Headline = props => {
       className={classnames({
         [styles.truncate]: props.truncate,
       })}
+      title={props.title || undefined}
```

Comment:
I would either introduce a function to clarify why the `undefined` fallback is used.

```js
// This function avoids empty strings as titles as browsers
// will show an empty title otherwise
const getOptionalTitle = title => title || undefined
```

```jsx
title={getOptionalTitle(props.title)}
```

This helps the next dev understand what's going on here.


Or, I would just do `title={props.title}`. It's the parent's fault if they screw this up and it's not something we should fix for them? We could add a warning in `propTypes` in case `title` is an empty string and encourage to pass `undefined` instead.

Reply:
@dferber90 yeah I would go for the second.

Explaining it is just explaining how react works...